src/components/campaigns/CampaignCard.tsx

```diff
@@ -118,14 +119,20 @@ type Props = { campaign: CampaignResponse }
 export default function CampaignCard({ campaign }: Props) {
   const classes = useStyles()
   const { t } = useTranslation()
+  const { publicRuntimeConfig } = getConfig()
   //const amounts = [20, 50, 100]
   //const [alignment, setAlignment] = React.useState<string | null>('left')
   // const handleAlignment = (event: React.MouseEvent<HTMLElement>, newAlignment: string | null) => {
   //   setAlignment(newAlignment)
   // }
+  const undefinedUrl = publicRuntimeConfig.API_URL + '/api/campaign-file/' + undefined
```

Comment:
This row is very uncommon, can you please share why you needed such variable name and structure??

Reply:
We need it to check if there is no valid picture url.

Reply:
It's going to be better to make a condition based on the result of 

```tsx
function campaignPicture(campaign): string | null {
  const background = campaign.campaignFiles.find((x) => x.role == 'background')
  if (background) {
    return background.id
  }
  return null
}
```

by encapsulating this functionality you can reuse it elsewhere and you don't have to do against checks undefined values